paper2remarkable/ui.py

```diff
@@ -184,17 +189,22 @@ def choose_provider(cli_input):
         Raised when the input *is* a valid url, but no provider can handle it.
 
     """
+
```

Comment:
With the change to the base `Provider` class suggested above, I believe these changes are no longer needed